config/createRollupConfig.js

```diff
@@ -0,0 +1,17 @@
+const cjs = require('@rollup/plugin-commonjs');
+const {resolve} = require('path');
+
+/**
+ *
+ * @param {string} path
+ */
+module.exports = function createRollupConfig(path) {
+  return {
+    input: resolve(path, './index.js'),
+    output: {
+      file: resolve(path, './index.esm.js'),
```

Comment:
This extension does not work for NodeJS' native esm implementation, the extension needs to be `.mjs` for NodeJS to run the file in ESM mode.


Right now when using NodeJS' native esm implementation, an error will be thrown when using `import` on any Just packages:
```
(node:55016) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
/tmp/just-esm-example/node_modules/just-camel-case/index.esm.js:40
export {stringCamelCase as default};
^^^^^^

SyntaxError: Unexpected token 'export'
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:196:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:183:25)
    at async Loader.import (node:internal/modules/esm/loader:178:24)
```